genie-web/src/main/java/com/netflix/genie/web/resources/handlers/GenieResourceHttpRequestHandler.java

```diff
@@ -72,6 +81,15 @@ public GenieResourceHttpRequestHandler(final DirectoryWriter directoryWriter, fi
         super();
         this.directoryWriter = directoryWriter;
         this.jobFileService = jobFileService;
+
+        try {
+            this.jobFileService.createJobDirectory(V4_MOCK_JOB_ID);
```

Comment:
Could also create empty `stdout` and `stderr` since it's kinda part of the contract that those files exist.
Not saying it's our API, but downstream crawlers may *assume* they exist, and they wouldn't be wrong normally.